scripts/generate_gauss_legendre_ref_data.py

```diff
@@ -26,7 +30,7 @@ def dump_reference_json():
             x,w = get_leggauss(order, a, b)
             data.append(dict(a=a,b=b,order=order,points=x.tolist(),weights=w.tolist()))
     print(len(data))
-    with open(path+"tests/reference_data/gauss_legendre_reference.ubjson",'wb') as f:
+    with open(path+outputs_path+"gauss_legendre_reference.ubjson",'wb') as f:
```

Comment:
Just a nitpick, but as long as we're changing this line, path joining in Python should really be done with `os.path.join` rather than this ugly inline-plus syntax.